src/functions/template-tags/link.php

```diff
@@ -483,23 +483,30 @@ function tribe_get_event_website_link( $event = null, $label = null, $target = '
 }
 
 /**
- * Get the link for the event website.
+ * Retrieves the formatted title for the event website link.
+ *
+ * This function fetches the event website title, which can be customized
+ * using the 'tribe_events_get_event_website_title' filter.
  *
  * @since 5.5.0
  *
- * @param null|int $post_id The event or event ID.
- * @return string  Formatted title for the event website link
+ * @param null|int $post_id Optional. The event post object or event ID. Defaults to null, which uses the current event ID.
```

Comment:
Since we're just describing the parameter, I don't think we need to go into the computation of the value if it's null.
```suggestion
 * @param null|int $post_id Optional. The event post object or event ID. Defaults to null.
```